### DISCUSSION TITLE
How to continue execution even after the step failed in cucumber 7.18.0

### DISCUSSION THREAD

**Discussion by ak270296** [cucumber/common#2181] [Q&A Java] [answered]
I am using cucumber 7.18.0 with selenium java here if the step fails in the test case the execution of the next steps does not continue. So I want to mark the failed step as failed and continue the execution. This logic exists in my old project where I use info.cukes dependencies.

**Comment by mpkorstanje** [comment, marked as answer]
Discussed here; https://github.com/cucumber/cucumber-jvm/issues/2904
